### ISSUE TITLE
I have erroneously sent Bitcoin to a Bitcoin Cash address. And now?

### ISSUE THREAD

**Issue by lucapsg** [bitpay/copay#7106]
I have erroneously sent Bitcoin to a Bitcoin Cash address. And now?

**Comment by JDonadio** [comment]
Please check this out: https://github.com/bitpay/copay/wiki/COPAY---FAQ#i-sent-bch-coins-into-a-btc-address-what-can-i-do-to-recover-my-bch-coins

**Comment by lucapsg** [comment]
Thanks for the advice but I can not.
The address of the recipient is a exchange.
Besides, I sent BTC to a BCH address, not the reverse.
How should I proceed?

**Comment by JDonadio** [comment]
Your bitcoins are in that BCH address, but if you cannot reach they you must contact to the exchange and ask they for your coins, explaining the situation.

**Comment by lucapsg** [comment]
Okay, I got them.
But since in general settings I did not enable "Bitcoin Cash" support, the software should not be able to verify if the address is correct?

**Comment by lucapsg** [comment]
Yesterday I contacted the exchange, asked me the TXID and then I did not hear it anymore.
1. Does it take so long to trace a transaction?
2. Is it possible that after a certain number of hours/days the transaction will be rejected by the network and the amount will return to the wallet? (I had set the fees to the minimum)
3. Is it normal that the new balance I should have is not completely available? For example: initial balance 1 BTC, sent 0.75, new balance 0.25 BTC **BUT** only 0.05 in "available" state. In the wallet the other 0.20 is in "**confirming**" state! 
Help me, I'm desperate !!!
